imsim/ccd.py

```diff
@@ -122,6 +123,35 @@ def buildImages(self, config, base, file_num, image_num, obj_num, ignore, logger
             rng = galsim.config.GetRNG(config, base)
             cosmic_rays.paint(image.array, rng, exptime=exp_time)
 
+        # Add header keywords for various values written to the primary
```

Comment:
This block is new functionality, i.e., writing info from the opsim meta dict object to the PHDU of the eimage file.  I think it's pretty straight-forward, but thought I should point it out anyway.

Reply:
The new bit just being that these values are now in the eimage header as well, rather than just the final final's headers, right?  That seems like a good change to me.